### DISCUSSION TITLE
MQTT_password bug?

### DISCUSSION THREAD

**Discussion by bartlauwers53** [arendst/Tasmota#14152] [Q&A] [answered]
Since when I started with Tasmota on my shelly devices about a year ago, I've used the same MQTT user and password to talk to my HA . in the beginning, these credentials worked fine. After issue with some module I've reset it and reconfigured it but it would not connect anymore. After some troubleshooting I gave up and blamed the device. 
Recently, I purchased new devices(shelly's) and flashed them with Tasmota 10.1.0. These devices also refused to connect. 
After downloading a tool to read the Tasmota's config, I found the value mqtt password(pwd_mqtt) te be messed up and containing the topic and full topic name and value. This might be caused because I have used special characters(!, &, ~, %, \ and / ), but as it used to work before, I expected to still be working. Is this a bug or are these characters not supported anymore?

**Comment by Jason2866** [comment, marked as answer]
Special characters are not supported (see docu).

**Reply by bartlauwers53** [reply]
Ah thanks for the quick answer. It did work before, so I assumed the same password would not cause issues after updates, but unfortunately, that is not the case.
